core/src/main/java/tc/oc/pgm/goals/ModeChangeGoal.java

```diff
@@ -11,6 +11,4 @@
   boolean isObjectiveMaterial(Block block);
 
   String getModeChangeMessage(Material material);
-
-  boolean isAffectedByModeChanges();
```

Comment:
Consider adding here `ImmutableSet<Mode> getModes` to replace the previous method